test/test_base_PowerModels_interface.jl

```diff
@@ -24,12 +24,12 @@ case5_dc_data = PM.replicate(case5_dc_data, 2)
 end
 
 @testset "PM with type extensions" begin
-    pm = PowerSimulations.build_nip_model(case5_data, DCPPowerModel)
-    JuMP.num_variables(pm.model) == 34
-    pm = PowerSimulations.build_nip_model(case5_data, ACPPowerModel)
-    JuMP.num_variables(pm.model) == 96
+    #pm = PowerSimulations.build_nip_model(case5_data, DCPPowerModel)
```

Comment:
These tests are different, one includes dc data and the other one doesn't

Reply:
both did not include dc data (case5_data) so it was repetitive, I can change it to be case5_dc_data... I kind of figured that was the intention but wasn't sure.